Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -90,6 +90,7 @@ public static class StreamingHelpers
             }
 
             var enableDlnaHeaders = !string.IsNullOrWhiteSpace(streamingRequest.Params) ||
+                                    streamingRequest.StreamOptions.ContainsKey("dlnaheaders") ||
```

Comment:
Are you certain that this works?

Reply:
![image](https://user-images.githubusercontent.com/7069165/144064468-03aa6d9e-cd5d-4c53-824d-701927184b21.png)
I just verified and parameters that aren't explicitly mapped will end up there.